File: prez/code/mem1.c
#include <stdio.h>
#define N (2<<22)
int main()
{
  int t[N]={0, 1}; // alloc ?
  float s = 0;
  for(int i = 0; i < N ; ++i) 
    s+= t[i];
  printf("%f", s);
  return 0;
}

File: prez/code/mem1.f90
program alloc
  integer, parameter :: N = ishft(2,22)
  call compute_sum()
contains
  subroutine compute_sum()
    real :: t(N)
    t(1:2) = [0,1]
    print *, sum(t)
  end subroutine compute_sum 
end program alloc
